pygmt/tests/test_info.py

```diff
@@ -29,6 +30,48 @@ def test_info():
     assert output == expected_output
 
 
+def test_info_path():
```

Comment:
We can use `pytest.mark.parametrize` to pass different parameters to the same function so that we don't have to duplicate the code. The `pytest.mark.parametrize` is a little complicated here, because `pathlib.PureWindowsPath` only works on Windows and `pathlib.PurePosixPath` doesn't work on Windows. 
```suggestion
@pytest.mark.parametrize(
    "table",
    [
        pathlib.Path(POINTS_DATA),
        pytest.param(
            pathlib.PureWindowsPath(POINTS_DATA),
            marks=pytest.mark.skipif(
                sys.platform != "win32",
                reason="PureWindowsPath is only available on Windows",
            ),
        ),
        pytest.param(
            pathlib.PurePosixPath(POINTS_DATA),
            marks=pytest.mark.skipif(
                sys.platform == "win32",
                reason="PurePosixPath is not available on Windows",
            ),
        ),
    ],
)
def test_info_path(table):
```